base/src/main/java/com/smartdevicelink/proxy/rpc/OnVehicleData.java

```diff
@@ -582,21 +582,37 @@ public void setCloudAppVehicleID(String cloudAppVehicleID){
     public String getCloudAppVehicleID(){
         return getString(KEY_CLOUD_APP_VEHICLE_ID);
     }
-
-    /**
-     * Sets a value for OEM Custom VehicleData.
-     * @param vehicleDataName a String value
-     * @param vehicleDataState a VehicleDataResult value
-     */
-    public void setOEMCustomVehicleData(String vehicleDataName, Object vehicleDataState){
-        setParameters(vehicleDataName, vehicleDataState);
-    }
-
-    /**
-     * Gets a VehicleData value for the vehicle data item.
-     * @return a Object related to the vehicle data
-     */
-    public Object getOEMCustomVehicleData(String vehicleDataName){
-        return getParameters(vehicleDataName);
-    }
-}
+
+    /**
+     * Sets a value for OEM Custom VehicleData.
+     * @param vehicleDataName a String value
+     * @param vehicleDataState a VehicleDataResult value
+     */
+    public void setOEMCustomVehicleData(String vehicleDataName, Object vehicleDataState){
+        setParameters(vehicleDataName, vehicleDataState);
+    }
+
+    /**
+     * Gets a VehicleData value for the vehicle data item.
+     * @return a Object related to the vehicle data
+     */
+    public Object getOEMCustomVehicleData(String vehicleDataName){
+        return getParameters(vehicleDataName);
+    }
+
+    /**
+     * Sets a value for StabilityControlsStatus.
+     * @param stabilityControlsStatus a StabilityControlsStatus value
+     */

```

Comment:
```suggestion
   /**
     * Sets the stabilityControlsStatus.
     *
     * @param stabilityControlsStatus See StabilityControlsStatus
     * @since SmartDeviceLink 7.0.0
     */

```